lib/runners/julia.js

```diff
@@ -12,7 +12,7 @@ module.exports.run = function run(opts, cb) {
       if (opts.setup) codeWriteSync('julia', opts.setup, juliaCodeDir);
       runCode({
         name: 'julia',
-        args: ['', ['push!(LOAD_PATH, "', juliaCodeDir, '", "frameworks/julia")'].join(""),
+        args: ['-P', ['push!(LOAD_PATH, "', juliaCodeDir, '", "frameworks/julia")'].join(""),
```

Comment:
`-P` is not deprecated? I thought you removed it because it was deprecated. Can you clarify?

The suggested fix I meant was this:
```js
runCode({
  name: 'julia',
  args: ['-q', solutionFile],
  options: {
    cwd: juliaCodeDir,
  }
});
```

Write `opts.solution` to a file and set `solutionFile` to the path of the file.
Use `-q` so that Julia banner is not printed.
Code in `opts.setup` should be accessible from the solution using `include` or `require`.

Please correct me if I'm misunderstanding.